fang/README.md

```diff
@@ -261,10 +290,10 @@ let mut queue = AsyncQueue::builder()
     .build();
 
 // Always connect first in order to perform any operation
-queue.connect(NoTls).await.unwrap();
+queue.connect().await.unwrap();
 ```
 
-As an easy example, we are using NoTls type. If for some reason you would like to encrypt Postgres requests, you can use [openssl](https://docs.rs/postgres-openssl/latest/postgres_openssl/) or [native-tls](https://docs.rs/postgres-native-tls/latest/postgres_native_tls/).
+Encryption is always used with crate `rustls`, if you want to not use encryption, you can issue us to re-export the sqlx feature with no encryption.
```

Comment:
Now that I think about, maybe it would be better to have an encryption feature that is enabled by default. This is because, if I remember correctly, features are supposed to add functionality, not remove it.

Reply:
I think `sqlx` does not provide a way to use it without encryption.

Maybe I should edit this comment

Reply:
We will add this option when we are able to bump `sqlx` version then.